src/components/Form.js

```diff
@@ -3,14 +3,15 @@ import injectSheet from 'react-jss'
 
 import H2 from 'components/H2'
 
-import { paper, shadow } from 'constants/styles/colors'
+import { paper, shadow, black, yellow } from 'constants/styles/colors'
+import { robotoBold } from '../constants/styles/fonts'
 
 const styles = {
 	formWrapper: {
-		backgroundColor: paper,
-		boxShadow: [[1, 1, 2, 2, shadow]],
+		backgroundColor: yellow,
+		border: '2px solid black',
```

Comment:
I know this is annoying but this isn't the `black` that's imported from `constants/styles/colors`. Because this is just a string, it doesn't know about the javascript constant `black`. To make that work, do `` `2px solid ${black}` ``